src/schemas/authSchemas.ts

```diff
@@ -2,6 +2,7 @@ import * as yup from 'yup'
 
 export const loginSchema = yup.object().shape({
   email: yup.string().email().required(),
+  password: yup.string().required('Enter a password'),
 })
```

Comment:
either use the default required message on both entries or have a custom message for them both